src/main/scala/com/codacy/client/bitbucket/v2/service/PullRequestServices.scala

```diff
@@ -28,18 +31,20 @@ class PullRequestServices(client: BitbucketClient) {
    *
    */
   def getPullRequestCommits(owner: String, repository: String, prId: Long): RequestResponse[Seq[SimpleCommit]] = {
-    val url = s"https://bitbucket.org/api/2.0/repositories/$owner/$repository/pullrequests/$prId/commits?pagelen=100"
+    val pullRequestsUrl = generatePullRequestsUrl(owner, repository)
+    val url = s"$pullRequestsUrl/$prId/commits?pagelen=100"
 
     client.executePaginated(Request(url, classOf[Seq[SimpleCommit]]))
   }
 
   private[this] def postNewComment(
       owner: String,
-      repo: String,
+      repository: String,
       prId: Int,
       values: JsObject
   ): RequestResponse[PullRequestComment] = {
-    val url = s"https://bitbucket.org/api/2.0/repositories/$owner/$repo/pullrequests/$prId/comments"
+    val pullRequestsUrl = generatePullRequestsUrl(owner, repository)
+    val url = s"$pullRequestsUrl/$prId/comments"
```

Comment:
Codacy found an issue: [The string literal '${ .. }/${ .. }/comments' appears 2 times in the file.](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)